src/js/tracing/reactnavigation.ts

```diff
@@ -101,7 +102,9 @@ export class ReactNavigationInstrumentation extends InternalRoutingInstrumentati
 
     if (this._options.enableTimeToInitialDisplay) {
       this._newScreenFrameEventEmitter = createSentryEventEmitter();
+      this._newFallbackEventEmitter = createSentryFallbackEventEmitter();
```

Comment:
I think `sentryEventEmitter` could be internally used in the `sentryFallbackEventEmitter`. To isolate the fallback logic and remove it from ReactNavigation.

Reply:
I think you want to move the fallback inside of sentryEventEmitter? if so, I moved it.

Reply:
I was thinking about something like this: https://github.com/getsentry/sentry-react-native/compare/test/deadline...kw-fallback-emitter-suggestion

Using the `sentryEventEmitter` inside of `sentryFallbackEventEmitter` and than use the `sentryFallbackEventEmitter` in `reactnavigation`.

Let me know what do you think about the suggestion.